bindings/wasm/src/account/types/cek_algorithm.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use identity::account_storage::CEKAlgorithm;
+use serde::Deserialize;
+use serde::Serialize;
+use wasm_bindgen::prelude::*;
+
+use crate::account::types::WasmAgreementInfo;
+use crate::error::Result;
+use crate::error::WasmResult;
+
+/// Supported CEK algorithms
+#[derive(Clone, Serialize, Deserialize)]
+#[wasm_bindgen(js_name = CEKAlgorithm, inspectable)]
+pub struct WasmCEKAlgorithm(CEKAlgorithm);
+
+#[wasm_bindgen(js_class = CEKAlgorithm)]
+impl WasmCEKAlgorithm {
+  /// ECDH-ES will be used as the content encryption key.
+  #[wasm_bindgen(js_name = ecdhES)]
+  pub fn ecdh_es(agreement: &WasmAgreementInfo) -> WasmCEKAlgorithm {
```

Comment:
Nit: would prefer `ECDH_ES` casing for consistency.

```suggestion
  #[wasm_bindgen(js_name = ECDH_ES)]
  pub fn ecdh_es(agreement: &WasmAgreementInfo) -> WasmCEKAlgorithm {
```